main/src/cgeo/geocaching/maps/mapsforge/v6/TapHandler.java

```diff
@@ -42,13 +42,16 @@ public synchronized void finished() {
         hitItems.clear();
     }
 
-    public void onLongPress(final Point tapXY) {
+    /** returns true if map item was hit */
```

Comment:
> returns true if map item was hit

Isn't it exactly the other way around? If no item was hit / if empty space was hit...

Reply:
you're right, will change the comment